### DISCUSSION TITLE
Is there anyway to copy link text instead of the link url?

### DISCUSSION THREAD

**Discussion by zhaocai** [hoothin/SearchJumper#51] [Ideas]
As asked in the title, is there a parameter like`%t` to get the link text? It is inconvenient to select the link text; Could be handy to drag the link and copy the text.

**Comment by hoothin** [comment]
`%s` 就是，選中文字代表文字，指向連結代表連結内容文字，但如果當前有匹配引擎，則取上次抓取到的關鍵詞。另外有個實驗性參數 `%n` 可以獲取指向元素的 title 或者 document 的 title。
你提問之前先自己試一下呀。

**Comment by zhaocai** [comment]
> `%s` 就是

我把简单的问题想复杂了，多谢了！

**Comment by zhaocai** [comment]
复制链接文本在最新的搜索酱版本失效了。操作步骤：左键抓取链接，放到下图的复制 `copy:%sr`图标上，什么都没有复制。
![image](https://github.com/hoothin/SearchJumper/assets/341473/66c87f90-5ce3-4250-8e7c-4aabb9667205)
